spec/spec_helper.rb

```diff
@@ -1,10 +1,9 @@
 # encoding: utf-8
 
-if RUBY_VERSION >= '1.9'
```

Comment:
No current CI, especially github actions has build in support for rubies smaller than 1.9.